docs/swarm/networking.md

```diff
@@ -0,0 +1,315 @@
+<!--[metadata]>
++++
+title = "Attach services to an overlay network"
+description = "Use swarm mode networking features"
+keywords = ["guide", "swarm mode", "swarm", "network"]
+[menu.main]
+identifier="networking-guide"
+parent="engine_swarm"
+weight=16
++++
+<![end-metadata]-->
+
+# Attach services to an overlay network
+
+Docker Engine swarm mode provides networking features to help you attach services to a user-defined network:
```

Comment:
Please remove this sentence. Currently swarm-mode doesnt support user-defined networks using any other driver other than `overlay driver`.